src/id-card/index.html

```diff
@@ -21,10 +21,10 @@
     <!-- development version, includes helpful console warnings -->
     <script src="https://unpkg.com/vue@next"></script>
 
-    <script src="assets/irma.js" defer></script>
-    <script src="assets/js/irma-functions.js" defer></script>
-    <script src="assets/js/ui-functions.js" defer></script>
-    <script src="assets/js/global.js">
+    <script src="../assets/irma.js" defer></script>
```

Comment:
Do we need to delete `./id-card/assets/irma.js` and `./id-card/assets/js/irma-functions.js` as you are loading them from root?

Reply:
Yes, we can delete them.